scripts/release_scripts/generate_release_updates.py

```diff
@@ -159,7 +159,12 @@ def create_group_for_next_release():
         'the release co-ordinator & QA Lead for next release. '
         'Please follow the release process doc: '
         '[Add link to release process doc] to ensure the release '
-        'follows the schedule. Thanks!\n' % (
+        'follows the schedule. Thanks!\n\n'
+        '3. Send the following message to oppia-dev:\n\n'
+        'Hi all, We have completed the release for current month. '
+        'This is a reminder to fill in the job requests here: '
+        'https://goo.gl/forms/XIj00RJ2h5L55XzU2 if you are planning '
+        'to run your job in the next release. Thanks!\n' % (
```

Comment:
Yes, I think we can do that. Should a week before be fine?

Reply:
I think it should be close to the release-cut date, so in general case it would be around 10-15 days and it worst case it can be a week. I would suggest having a template <[Add upcoming release cut date]> in the message for the last date. Having a constant time can delay the upcoming release cut in the case where the current release is already not on time.  

[Also, I realized the QA co-ordinator use to send an email to oppia-dev about a week before the release cut about the upcoming release and release cut-date, maybe this can be a part of that message? (not sure!)]

Reply:
I think having the previous release coordinator send the email about who the next release/QA coordinator is, when the release cut date is, and this reminder, is a great way to wrap up the release, and fits in well with the separate email that celebrates the launch of the previous one.

Maybe we could modify the process to have this be the procedure instead?

Also, re Sandeep's point, I think we'll also need to give guidance to the release coordinator on how to pick a date. Perhaps say "choose a deadline that's at least 7 days before the next release cut"?

Reply:
> Maybe we could modify the process to have this be the procedure instead?

Sorry, I did not understand this. What do you mean by modify the process? We already follow this script in the release process.

> Also, re Sandeep's point, I think we'll also need to give guidance to the release coordinator on how to pick a date. Perhaps say "choose a deadline that's at least 7 days before the next release cut"?
Done.

Reply:
Sorry, scratch my earlier comment. I misread the email message. Everything LGTM. Thanks!